### PR TITLE
Fix minor typo

### PR DESCRIPTION
**Pull request by harikrishnana2021**
Fixed minor typo

## Description
N/A

### CODE REVIEW

File: packages/gatsby-transformer-xml/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # gatsby-transformer-xml
 
-Parses XML files. It supports also attributes
+Parses XML files. It supports attributes also
```
Comment (wardpeet) [inline packages/gatsby-transformer-xml/README.md]: This might sound a bit better
```suggestion
Parses XML files. It also supports attributes
```

**Comment by harikrishnana2021** [comment]
\cc @wardpeet

**Comment by wardpeet** [comment]
@harikrishnana2021 we reall appreciate the time you're taking fixing spelling errors. Could you be so kind to bundle all typos in one PR?

**Pull request merged**